modules/data-streams/build.gradle

```diff
@@ -43,4 +43,5 @@ if (BuildParams.isSnapshotBuild() == false) {
 
 tasks.named("yamlRestCompatTestTransform").configure({ task ->
   task.skipTest("data_stream/10_basic/Create hidden data stream", "warning does not exist for compatibility")
+  task.skipTest("data_stream/150_tsdb/TSDB failures go to failure store", "timestamp values are invalid")
```

Comment:
We can mute the test here, or fix the timestamps on the 8.x branch to match here

Reply:
#116708